infra/terraform/terraform.go

```diff
@@ -260,8 +261,8 @@ var (
 )
 
 // installerCommand waits for the installer tarball to download, unpacks it and launches the installation
```

Comment:
The comment is off now - `installerCommand returns a shell command to fetch installer tarball, unpack it and launch the installation`